grumblr/app/assets/javascripts/application.js

```diff
@@ -13,4 +13,25 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
+//= require underscore
+//= require backbone
+//= require handlebars.runtime
+//= require_tree ./templates
+//= require_tree ./backbone/routers
+//= require_tree ./backbone/models
+//= require_tree ./backbone/collections
+//= require_tree ./backbone/views
 //= require_tree .
+
+$(document).ready(loadGrumbles);
+function loadGrumbles(){
+  mainCollection = new GrumblesCollection();
+  mainCollection.fetch().done(function(){
+    mainCollection.each(function(currentModel){
+      var currentView = new GrumbleView({
+        model: currentModel
+      });
+      $('#grumble').append(currentView.$el);
+    })
+  })
+}
```

Comment:
this sort of thing is kind of what the interaction between chad/ali/adam this morning was describing.  While this works, it's not integrating a list view. I think this implementation is totally fine for this application; however, when we start to scale our apps and make them more modular. It'll be really nice to have one constructor function that handles all the behaviors in the CRUD of a collection. It's also nice that we have a place where we can add functionality within list views. Additionally it keeps logic and functionality out of the `app.js` file.